platforms/allwinner-d1/d1-core/src/ccu.rs

```diff
@@ -20,10 +20,10 @@ pub struct Ccu {
 
 /// Trait to be implemented for module clocks that can be gated and reset
 pub trait BusGatingResetRegister {
-    /// Enable or disable the clock reset bit
-    fn gating(ccu: &mut CCU, pass: bool);
     /// Enable or disable the clock gating bit
-    fn reset(ccu: &mut CCU, deassert: bool);
+    fn gating(ccu: &mut CCU, pass: bool);
+    /// Enable or disable the clock reset bit
+    fn reset(ccu: &mut CCU, assert: bool);
```

Comment:
We didn't really get them backwards, but the initial API wasn't very intuitive.
When you passed `true` to reset, it *de-asserted* the reset (because that's what writing `1` in the register does).
I switched it around now, but using enums is probably the better idea.